packages/wallets/src/features/cashier/modules/TransactionStatus/TransactionStatus.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback } from 'react';
+import { useActiveWalletAccount } from '@deriv/api';
+import { WalletButton, WalletText } from '../../../../components/Base';
+import { Loader } from '../../../../components/Loader';
+import Warning from '../../../../public/images/warning.svg';
+import { CryptoTransaction } from './components/CryptoTransaction';
+import useRecentTransactions from './hooks/useRecentTransactions';
+import './TransactionStatus.scss';
+
+type TTransactionStatus = {
+    transactionType?: 'deposit' | 'withdrawal';
```

Comment:
Is this prop value coming from API. If it is, can we type it better ?

Reply:
Have done using `THooks`